shunting-yard-common/src/test/java/com/hotels/shunting/yard/common/io/jackson/DeserializerWithJsonInputTest.java

```diff
@@ -61,71 +73,104 @@
-
-  private final static String CREATE_TABLE_EVENT = "\"Message\" : \"{\\\"protocolVersion\\\":\\\"1.0\\\",\\\"eventType\\\":\\\"CREATE_TABLE\\\",\\\"dbName\\\":\\\"some_db\\\",\\\"tableName\\\":\\\"some_table\\\"}\"";
-  private final static String DROP_TABLE_EVENT = "\"Message\" : \"{\\\"protocolVersion\\\":\\\"1.0\\\",\\\"eventType\\\":\\\"DROP_TABLE\\\",\\\"dbName\\\":\\\"some_db\\\",\\\"tableName\\\":\\\"some_table\\\"}\"";
-  private final static String INSERT_EVENT = "\"Message\" : \"{\\\"protocolVersion\\\": \\\"1.0\\\",\\\"eventType\\\": \\\"INSERT\\\",\\\"dbName\\\": \\\"some_db\\\",\\\"tableName\\\": \\\"some_table\\\",\\\"files\\\": [\\\"file:/a/b.txt\\\",\\\"file:/a/c.txt\\\"],\\\"fileChecksums\\\": [\\\"123\\\",\\\"456\\\"],\\\"partitionKeyValues\\\": {\\\"col_1\\\": \\\"val_1\\\",\\\"col_2\\\": \\\"val_2\\\", \\\"col_3\\\": \\\"val_3\\\"}}\"";
+  private String addPartitionEvent;
+  private String alterPartitionEvent;
+  private String createTableEvent;
+  private String dropPartitionEvent;
+  private String insertEvent;
+  private String alterTableEvent;
+  private String dropTableEvent;
 
   private static final Map<String, String> PARTITION_KEYS_MAP = ImmutableMap
       .of("col_1", "string", "col_2", "integer", "col_3", "string");
   private static final List<String> PARTITION_VALUES = ImmutableList.of("val_1", "val_2", "val_3");
   private static final List<String> OLD_PARTITION_VALUES = ImmutableList.of("val_4", "val_5", "val_6");
+
+  private static final Map<String, String> TABLE_PARAM_MAP = ImmutableMap.of("param_1", "val_1", "param_2", "val_2");
+
   private final static String TEST_DB = "some_db";
   private final static String TEST_TABLE = "some_table";
+  private final static String TEST_TABLE_LOCATION = "s3://table_location";
+  private final static String OLD_TEST_TABLE_LOCATION = "s3://old_table_location";
+  private final static String PARTITION_LOCATION = "s3://table_location/partition_location";
+  private final static String OLD_PARTITION_LOCATION = "s3://table_location/old_partition_location";
+
+  @Before
+  public void init() throws IOException {
+    addPartitionEvent = new String(Files.readAllBytes(dataFolder.getFile("add_partition.json").toPath()))
+        .replace("\n", "");
+    alterPartitionEvent = new String(Files.readAllBytes(dataFolder.getFile("alter_partition.json").toPath()))
+        .replace("\n", "");
+    dropPartitionEvent = new String(Files.readAllBytes(dataFolder.getFile("drop_partition.json").toPath()))
+        .replace("\n", "");
+    createTableEvent = new String(Files.readAllBytes(dataFolder.getFile("create_table.json").toPath()))
+        .replace("\n", "");
+    insertEvent = new String(Files.readAllBytes(dataFolder.getFile("insert_table.json").toPath())).replace("\n", "");
```

Comment:
All other variables are named <action>table/partitionEvent. Can this be named `insertTableEvent` please? :)

Reply:
Yeah. InsertTableEvent doesn't sound right to me :P